content/posts/2022-08-17-ocp-on-m1.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Running ocp on M1 machines"
+date: 2022-08-17T08:58:10+05:30
+---
+
+From the 2.4.1 release, CRC has been running on macOS aarch64 M1 CPUs.
```

Comment:
With the release of CRC 2.4.1 it was possible to run on Apple's latest offering of M1-based machines.